src/transformStateWithClones.js

```diff
@@ -65,7 +65,53 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const arrOfState = [];
+  const cloneState = { ...state };
+
+  for (const indX of transforms) {
+    let newClone = {};
+
+    switch (indX.operation) {
+      case 'addProperties':
```

Comment:
```suggestion
      case 'addProperties': {
        ....
      }
```
на майбутнє, краще вікрива блок коду